ivadomed/config/config_new_loader.json

```diff
@@ -0,0 +1,88 @@
+{
+    "command": "train",
+    "gpu": 0,
+    "log_directory": "/home/mhbourget/ivadomed/log_directory/new_loader",
```

Comment:
I'd use a relative path such as
``` "log_directory": "log_directory/new_loader",```

that way it's easier for others to use it

Reply:
Fixed in e5eb063